packages/x-date-pickers-pro/src/internal/hooks/useRangePickerInputProps.ts

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
-import { PickersInputLocaleText } from '@mui/x-date-pickers';
+import { PickersInputLocaleText } from '@mui/x-date-pickers/locales';
```

Comment:
I think it would make sense
But I also think some people may be importing it today from `'@mui/x-date-pickers/locales'`
And we can't export it from both since the root import re-export both of those folders.
So we would have to remove it from it's current folder.

That's why I did not do it for now.